### DISCUSSION TITLE
Does Chocolatey collect and publish stats on 32-bit vs. 64-bit installations?

### DISCUSSION THREAD

**Discussion by mislav** [chocolatey/choco#2178] [General] [answered]
When viewing a Chocolatey package, download stats for that package are available per package version.

Does Chocolatey also keep some stats about the machine that requested the installation, such as whether the system was 32-bit or 64-bit?

As publishers of software that is included in Chocolatey, we would like to know how many of Windows users are on 32-bit Windows to help gauge how much, if at all, should we invest in maintaining the 32-bit version of the software. Thank you!

**Comment by pauby** [comment, marked as answer]
On the backend end, Chocolatey records when a package is downloaded from chocolatey.org. So, as an example it will record when `google-chrome` is downloaded. 

Within that package the installer that is needed (x86 / x64) is downloaded, in this case from Google. Chocolatey doesn't 'phone home' so doesn't record any of that information.

**Reply by mislav** [reply]
Thank you!
